README.md

```diff
@@ -232,6 +232,20 @@ bh.match('button', function(ctx) {
 });
 ```
 
+## Infinite loop detection
+
```

Comment:
Можно как-то так сказать:
The `enableInfiniteLoopDetection` method allows you to enable or disable the infinite loop detection process.

*Note* Enable the infinite loop detection in debugging mode only, because it slows down a template engine application.